src/validation/EmailValidator.java

```diff
@@ -0,0 +1,71 @@
+package validation;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmailValidator {
+
+	private String allowedSpecialSymbols = "~!@#$%^&*_+=-`'./{}";
+	private List<String> tldList;
+	
+	public boolean hasAtSign(String string) {
+		if (string.indexOf("@") == -1) return false;
+		return true;
+	}
+
+	public boolean hasInvalidCharacters(String string) {
+	    if (string.contains("..")) return true;

```

Comment:
I don't understand the purpose of this if.

Reply:
Also return should be on a separate line for better readability and enclosed with {}.